modules/structs/user_app.go

```diff
@@ -11,11 +11,13 @@ import (
 // AccessToken represents an API access token.
 // swagger:response AccessToken
 type AccessToken struct {
-	ID             int64    `json:"id"`
-	Name           string   `json:"name"`
-	Token          string   `json:"sha1"`
-	TokenLastEight string   `json:"token_last_eight"`
-	Scopes         []string `json:"scopes"`
+	ID             int64     `json:"id"`
+	Name           string    `json:"name"`
+	Token          string    `json:"sha1"`
+	TokenLastEight string    `json:"token_last_eight"`
+	Scopes         []string  `json:"scopes"`
+	Created        time.Time `json:"created_at"`
+	Updated        time.Time `json:"updated_at"`
```

Comment:
Maybe a new name is better for `updated_at`. How about `last_use`?

Reply:
I agree on the name not being good. In my PR description, I suggest _last_used_at_ (consistent with _created_at_). I was wondering why the field is called _Updated_ internally? Is the data for the token changed when it is used??

Reply:
When we have decided on a name for the new field, I will update my PR for the SDK here: https://gitea.com/gitea/go-sdk/pulls/707

Reply:
I updated the field name to _last_updated_at_.

Reply:
It's a legacy story.